projects/aca-content/src/lib/store/reducers/app.reducer.ts

```diff
@@ -173,11 +170,6 @@ function showInfoDrawerPreview(state: AppState) {
   return newState;
 }
 
-const toggleDocumentDisplayMode = (state: AppState) => ({
-  ...state,
-  documentDisplayMode: state.documentDisplayMode === 'list' ? 'gallery' : 'list'
```

Comment:
Does AppState still requires documentDisplayMode field? I can't find any reading usages after that change so maybe it can be removed?

Reply:
does not require, I will check and cleanup... this was some old demo and have not been used since implementation